Cargo.toml

```diff
@@ -104,6 +104,8 @@ bevy_core_pipeline = [
   "bevy_internal/bevy_core_pipeline",
   "bevy_asset",
   "bevy_render",
+  "bevy_pbr",
```

Comment:
I don't like pulling in `bevy_pbr` here. What's causing the failure?

Reply:
> which wants bevy_pbr for Opaque3dPrepass bindings initialization

Reply:
Let me pull in rendering folks, but I think the correct solution is to actually move that code.

Reply:
I said about [this initialization](https://github.com/bevyengine/bevy/blob/ecdd1624f302c5f71aaed95b0984cbbecf8880b7/crates/bevy_pbr/src/prepass/mod.rs#L151) in description

Reply:
> I think the correct solution is to actually move that code.

Which code are you proposing to move? Core3dPlugin from bevy_core_pipeline to bevy_pbr?